sdk/core/azure-core/src/main/java/com/azure/android/core/http/interceptors/ScrubEtagInterceptor.java

```diff
@@ -0,0 +1,40 @@
+package com.azure.android.core.http.interceptors;
```

Comment:
This interceptor is unique to Storage only, so we should put it in that package.

Reply:
See above comment